me.kozec.syncthingtk.yml

```diff
@@ -0,0 +1,42 @@
+app-id:              me.kozec.syncthingtk
+
+runtime:             org.gnome.Platform
+runtime-version:     '3.28'
+sdk:                 org.gnome.Sdk
+
+command:             syncthing-gtk
+rename-desktop-file: syncthing-gtk.desktop
+rename-icon:         syncthing-gtk
+
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland
+  - --socket=wayland
+  # Folders for syncing can be anywhere
+  - --filesystem=host
+  # Network access for sync
+  - --share=network
+
+modules:
+  - python2-pycairo.json
+  - python2-pygobject.json
+  - python2-setuptools_scm.json
+  - python2-python_dateutil.json
+  - python2-bcrypt.json
+
+  - name: syncthing-gtk
+    buildsystem: simple
+    build-commands:
+      - python2 setup.py build
+      - python2 setup.py install --prefix=/app --optimize=1
+      - install -Dm644 -t /app/share/appdata me.kozec.syncthingtk.appdata.xml
+    sources:
+      - type:   archive
+        url:    https://github.com/syncthing/syncthing-gtk/archive/v0.9.4.tar.gz
+        sha256: 586dc60460973c2664cee9f0fe6360b635a5b04278924f61c4bc9041723549f2
+
+      - type:   file
+        path:   me.kozec.syncthingtk.appdata.xml
```

Comment:
AFAIK syncthing build system is selfcontained so building it should be trivial (we have golang sdk extension too).

Reply:
I’ll look into it.